SQL_practice.sql

```diff
@@ -0,0 +1,87 @@
+--1
+SELECT * FROM patients
+where province_id is "ON";
+
+
+
+-- 2
+select Pr.province_name, count(Pa.patient_id) as 'Total count of patients'
```

Comment:
Please do not use spaces as a separator for aliases. Also they should be shorter, e.g. total_count